hedyorg/hedy
# [FIX] Don't lock level on quiz threshold when there is no quiz

**Description**
In this PR we fix a bug where level 15 would be blocked because there is no quiz in level 14. When a threshold is set a user would never be able to get it and unlock the next level. We fix this by checking if there is a quiz, otherwise unlock the level anyway.

**Fixes**
This PR fixes #3741

**How to test**
A bit of a hassle but... set a quiz threshold and make sure you have quiz score getting this threshold for all levels. Then verify that you can still go to level 15 even without achieving a score in level 14 (as there is no quiz there).
